gateway/src/chains/polygon/polygon_tokens_mumbai.json

```diff
@@ -4,20 +4,56 @@
     {
       "symbol": "WMATIC",
       "chainId": 80001,
-      "address": "0x9c3c9283d3e44854697cd22d3faa240cfb032889",
```

Comment:
Please remove the changes to addresses unless changes are really needed. This prevents potential security issues -thanks